shared/tests/utils/test_s3_client.py

```
@@ -0,0 +1,267 @@
+        yield mock_boto3, mock_client
+        # Cleanup
+        if "boto3" in sys.modules:
+            del sys.modules["boto3"]
+        if "botocore" in sys.modules:
+            del sys.modules["botocore"]
+        if "botocore.config" in sys.modules:
+            del sys.modules["botocore.config"]
+
+    def test_upload_bytes_success(self, s3_config, mock_boto3_module):
+        """Test successful bytes upload."""
+        mock_boto3, mock_client = mock_boto3_module
```

Ruff (0.14.14)

[warning] Unpacked variable `mock_boto3` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)